CHANGELOG.rst

```diff
@@ -1,3 +1,14 @@
+DEV
+----
+
+Improvements:
+
+* return created subscription from Yola.create_subscription (5f7aba284e4678fa10bcc6482c8f04168acc831c)
```

Comment:
> the changelog entry is added together with the feature (in a DEV section, for example)

Yeah. I would prefer that to linking to the commit. That's not what happened here, but I don't want to set this precedent. I'll remove the commit links.